application-example.conf

```diff
@@ -89,45 +137,35 @@ kafka {
 
   maxRequestSize: 250000000
 
-}
+  securityProtocol: "SSL"
 
-worker {
+  consumerPropertyFile: ".../consumer-conf/consumer.properties"
 
-  # maximum number of times across all workers that an attempt should be made to process any given document;
-  # when this property is not provided, retries will not be tracked and no limit will be imposed;
-  # this property requires that zookeeper is available at the specified zookeeper.connectString
-  #maxRetries: 2
+  producerPropertyFile: ".../consumer-conf/producer.properties"
 
-  # number of worker threads to start for each pipeline when running lucille in local / single-JVM mode
-  threads: 2
+  documentDeserializer: # TODO
 
-  # maximum time to spend processing a message, after which the worker will assume a problem and shut down
-  # NOTE: not supported while com.kmwllc.lucille.core.Worker and PipelineWorker are being reconciled
-  #maxProcessingSecs: 600 # 10 minutes
-}
+  documentSerializer: # TODO
 
-publisher {
+  events: # TODO
```

Comment:
true is the default
can set to false to indicate that "events" (indicating whether a document succeeded or failed indexing) should not be sent as Kafka message
a setting of false would be used in a context where Lucille is running without a Runner/Publisher that would receive events and determine when the run is complete; in this context, there might be a WorkerIndexer running that reads documents directly from a given Kafka topic and sends them to the search engine without tracking successes or failures

Reply:
Thanks! Addressed in new commit